docs/getting-started/install-cent-node.md

```diff
@@ -69,52 +97,35 @@ Once you have installed all the necessary packages, follow these steps to instal
   -a 8082 -p 38204
   ```
 
-  This command automatically adds the Identity keys to the `config.yaml` file.
+  This command automatically creates an Identity and keys needed and generates the `config.yaml` file required to run the node.
   Replace the `_KEY-FILE_` with the key file you obtained when creating the Ethereum account and `_PATH-TO-CONFIG-FILE_` with the location where you want the `config.yaml`  file to be stored.
 
-3. Run the Centrifuge Node using the `config.yaml` file you created:
+4. Run the Centrifuge Node using the `config.yaml` file you created:
 
   ```bash
   $ centrifuge run -c /PATH-TO-CONFIG-FILE/config.yaml
   ```
   Replace the `_PATH-TO-CONFIG-FILE_` with the location of the `config.yaml` file.
 
-### Installing the Ethereum Rinkeby Node
+#### Additional custom configuration
+* Configure node under NAT
 
-Rinkeby is a testnet
+If you want your node to be reachable outside of your private network you will need to manually specify the External IP where the node will be available:
+```yaml
+p2p:
+  externalIP: "100.111.112.113"
+``` 
 
-**Resource Requirements**
-
-* 4 Gigabyte memory
-  * Syncing mode uses around 3 Gigabyte
-  * Normal sync uses around 400 Megabyte
-* 2 cores
-  * Syncing mode uses about 2-3 cores
-  * Normal mode uses about 0.2 cores
-* Allocated disk space of at least 100 Gigabyte
-  * Sync mode uses about 1 Gigabyte
-  * Full node requires about 50 Gigabyte
-* Syncing will take about 1-2 hours. This can vary depending on the resource availability of the full node(s).
-  * Optionally, you can use SSDs speed it up.
+* Configure notification webhook (for incoming data from other peers)
 
-
-1. To run Ethereum node in the light mode as a binary, install `geth`. Installation instructions are available at [https://github.com/ethereum/go-ethereum/wiki/Installation-Instructions-for-Ubuntu](https://github.com/ethereum/go-ethereum/wiki/Installation-Instructions-for-Ubuntu).
-
-2. Set the following environment variables in the `config.yaml` file:
-
-  * RPC_PORT = 8545
-  * WS_PORT = 8546
-  * ETH_DATADIR = $HOME/Ethereum/Library/rinkeby
-
-
-3. Start the `geth` process:
-
-  ```bash
-  $ geth --rinkeby --light --rpc --rpcapi db,eth,net,web3,txpool --ws \
-  --wsorigins "*" --wsapi db,eth,net,web3,txpool > /tmp/geth.log 2>&1 &
-   ```
-
-You now need to wait for about 1-2 hours depending on the resources under the P2P connected network for the local node to sync up with the Rinkeby network.
+We provide a way to your upstream systems to receive events whenever a new document has been shared with your node, and we do it through WebHooks:
```

Comment:
To receive an event when a new document has been shared with your node, set your WebHook endpoint in the `config.yaml` file: